model_analyzer/config/generate/base_model_config_generator.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, config, model, client, variant_name_manager,
         model: The model to generate ModelConfigs for
         client: TritonClient
         variant_name_manager: ModelVariantNameManager
-        default_only: Bool 
```

Comment:
We don't want all of these whitespace changes. I assume it is VI adding it.  Hopefully if you format with yapf it will go away?

Reply:
I ran yapf, but it didn't change back. I think there was a trailing whitespace in the original code and my vi setup automatically deletes them. I'll see if I can turn off the feature.